python/lsst/multiprofit/fitting/fit_psf.py

```diff
@@ -635,11 +635,14 @@ def fit(
                 if column:
                     row[f"{prefix}{column}"] = True
                     logger.info(
-                        f"{id_source=} ({idx}/{n_rows}) PSF fit failed with not unexpected" f" exception={e}"
+                        f"{id_source=} ({idx}/{n_rows}) PSF fit failed with known" f" exception={e}"
```

Comment:
Is there a good reason to not just have a single `f` string here? I.e.:
```
f"{id_source=} ({idx}/{n_rows}) PSF fit failed with known exception={e}"
```

Reply:
Not sure what happened there... anyway, fixed.